14/index.php

```diff
@@ -116,14 +116,12 @@
   <li>Open a .kmp file with Keyman Config (#3183)</li>
   <li>Now supports Ubuntu 20.10 (Groovy) (#3876)</li>
   <li>Improved user interface</li>
-  <li>Improved support for KDE, Gnome, Fedora, Arch Linux</li>
```

Comment:
The work for Fedora isn't finished yet (and I don't know when it will be finished).